apt_installer/templates.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 Google Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"text/template"
+)
+
+var DOCKERFILE_TMPL *template.Template
+
+func init_templates() {
+	var err error
+
+	DOCKERFILE := `FROM {{.BaseImage}}
```

Comment:
This code is short enough now that it's probably easier to just move this into main.go.